api/enki.py

```diff
@@ -233,8 +228,6 @@ def reaper_request(self, identifier):
                 replace=ReplacementPolicy.from_license_source(self._db)
             )
 
-            self._db.commit()
```

Comment:
@leonardr Just to confirm, the commit should be called once we've iterated over every ID, right? I'm assuming that's the case since I see no change to the DB if I kill the script after only a few dozen iterations.